<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>569</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Polaczenie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Połączenie</string>
    </property>
    <widget class="QPushButton" name="pushButtonSearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Szukaj</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLink">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Połącz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClose">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rozłącz</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDevices">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Logi">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>391</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Logi</string>
    </property>
    <widget class="QTextEdit" name="textEditLogi">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyczyść</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>641</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Nowa wiadomość</string>
    </property>
    <widget class="QSpinBox" name="spinBox_ID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_ID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_B0">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_B1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_B2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_B3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_B4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_B5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_B6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_B7">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_B0">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bajt 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_B1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bajt 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_B2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bajt 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_B3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bajt 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_B4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bajt 4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_B5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bajt 5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_B6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bajt 6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_B7">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bajt 7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSendMsg">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wyślij</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Filtry</string>
    </property>
    <widget class="QSpinBox" name="spinBox_F0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_F2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_F1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_F3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFilterSet">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ustaw</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFilterOff">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wyłącz</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>comboBoxDevices</tabstop>
  <tabstop>pushButtonLink</tabstop>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>textEditLogi</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButtonSendMsg</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>textEditLogi</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
